public/assets/sass/mixins/_toast-notification.scss

```diff
@@ -0,0 +1,44 @@
+.toast {
+  position: fixed;
+  top: 0;
+  z-index: 2;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+
+  transform: translateY(100%);
+  opacity: 0;
+
+  transition: opacity 500ms, transform 500ms;
+
+  &.visible {
+    transform: translateY(0);
+    opacity: 1;
+  }
+
+  .toast-body {
+    margin: 28px;
+    padding: 20px 24px;
+    border-radius: 4px;
+    width: 20%;
+
+    &.success {
+      background-color: lightgreen;
```

Comment:
I love how you designed the modal!  Would it be possible to use the Mathsoc colour scheme (i.e. some combination of white and mathsoc pinks) rather than this green?  It would help to keep things visually consistent.